docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,131 @@
+		- set the system to SYS_4: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_4`
+	- else (the climate zone is_CZ_3b_3c_and_4_to_8): `else:`
+		- set the system to SYS_3: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_3`
+				
+- G3.1.1d "For laboratory spaces in a building having a total laboratory exhaust rate greater than 15,000 cfm, use a single system of type 5 or 7 serving only those spaces.  The lab exhaust fan shall be modeled as constant horsepower reflecting constant volume stack discharge with outdoor air bypass."
+- select assigned system type based on total building area and number of floors - for less than 5 floors and less than 150,000 sf we use system 5, anything else is system 7.
+- if the building is less than 6 floors, and < 150,000 sf: `if( num_floors < 6 && area < 150000 ):`
+	- then the system type is SYS-5: `G3_1_1d_expected_system_type = SYS_5`
+- all other cases: `else:`
+	- the system type is SYS-7: `G3_1_1d_expected_system_type = SYS_7`
+- loop through the zones_and_systems to see if any of the zones meets G3.1.1.d: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1d to determine if this zone meets this requirement: `if does_zone_meet_G3_1_1d(B_RMI,zone,zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] == YES):`
+		- change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1d"` 
+		- set the system to SYS_5 or SYS_7: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = G3_1_1d_expected_system_type`
+
+- G3.1.1e "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+- loop through the zones_and_systems to see if any of the zones meets this requirement: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1e to determine whether to change the system type (we are using != NO instead of YES, because the function can return MAYBE.  Here we will assign any "maybe's to the new system type.  The rule itself will deal with the uncertainty of vestibules using the function is_zone_a_vestibule): `if does_zone_meet_G3_1_1e(P-RMI,B-RMI,zone) != NO:`
+		- - change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1e"`
+		- choose system 9 or 10 based on climate zone: `if is_CZ_0_to_3a():`
+			- set the system to SYS_10: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_10`
+		- else (the climate zone is_CZ_3b_3c_and_4_to_8): `else:`
+			- set the system to SYS_9: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_9`
+
+
+- G3.1.1f "If the baseline HVAC system type is 9 or 10, use additional system types for all HVAC zones that are mechanically cooled in the proposed design."
```

Comment:
Prehaps we should check for G3.1.1f before we do G3.1.1e to avoid havings zones that met exception G3.1.1e re-considered for exception (f). But also OK to keep the current order because the outcome will still be correct.

Reply:
I would like to keep the order because this is the order specified in Section G3.1.1